Makefile

```diff
@@ -1,6 +1,9 @@
 build:
 	go build -mod=vendor -ldflags "-X k8s.io/client-go/pkg/version.gitCommit=$$(git rev-parse HEAD) -X k8s.io/client-go/pkg/version.gitVersion=v1.0.0+$$(git rev-parse --short=7 HEAD)" -o bin/insights-operator ./cmd/insights-operator
 
+build-debug:
```

Comment:
This stuff should probably go into a separate pull request, or else someone could complain about this later.

Reply:
Yeah, it makes sense, thanks.